web/source/kmwosk.ts

```diff
@@ -915,7 +915,7 @@ if(!window['keyman']['initialized']) {
 
         switch(code) {
           case osk.keyCodes['K_BKSP']:  //Only desktop UI, not touch devices. TODO: add repeat while mouse down for desktop UI
-            kbdInterface.output(1, keymanweb.domManager.getLastActiveElement(), "");
+            this.keyman.interface.defaultBackspace();
```

Comment:
This was breaking 'K_BKSP' on Android due to JS exceptions of `interface` being undefined.
`kbdInterface.defaultBackspace();` worked for me.

Maybe the comment about "Only desktop UI, not touch devices" can also be removed.

Reply:
Right, because the OSK hasn't properly been refactored yet.  Yeah, that gets 'fun' to manage at times.  Good catch.